pyisy/connection.py

```diff
@@ -146,7 +146,7 @@ async def request(self, url, retries=0, ok404=False, delay=0):
             ) as res:
                 if res.status == HTTP_OK:
                     _LOGGER.debug("ISY Response Received.")
-                    results = await res.text()
+                    results = await res.text(encoding="utf-8")
```

Comment:
Do we need an ignore here as well?

Reply:
Doesn't look to be an option. 

https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientResponse.text

Alternative would be `res.content.decode(encoding="utf-8", errors="ignore")` but content looks to be a different animal on `aiohttp` vs `requests`. I'm thinking it's actually the auto-detection of the encoding that's failing, not actually bad `utf-8`, so by forcing the encoding, it'll fix the issue.

Thoughts?

Reply:
```
    async def text(self,
                   encoding: Optional[str]=None, errors: str='strict') -> str:
        """Read response payload and decode."""
        if self._body is None:
            await self.read()

        if encoding is None:
            encoding = self.get_encoding()

        return self._body.decode(encoding, errors=errors)  # type: ignore
```

It looks like `res.text(encoding="utf-8", errors="ignore")` should work based on the code ^^

Reply:
Now you're just showing off........

Will fix it. Need to fix msg.data in the websocket too -- it's already decoded.